contracts/QuestFactory.sol

```diff
@@ -89,6 +97,19 @@ contract QuestFactory is Initializable, OwnableUpgradeable {
         return address(0);
     }
 
+    function grantCreateQuestRole(address account_) public {
```

Comment:
no need to make this or `revokeCreateQuestRole` onlyAdmin because only the `DEFAULT_ADMIN_ROLE` can grant or revoke roles.